File: Counting elements in two arrays.cpp
class Solution
{
  public:
    vector<int> countEleLessThanOrEqual(int arr1[], int arr2[], 
                                 int m, int n)
    {
        sort(arr2,arr2+n);
        vector<int>result;
        for(int i=0;i<m;i++)
        {
            auto y=upper_bound(arr2,arr2+n,arr1[i])-arr2;
            result.push_back(y);
        }
        return result;
    }
};

File: Counting elements in two arrays.py
from bisect import bisect_right
class Solution:
    def countEleLessThanOrEqual(self,arr1,n1,arr2,n2):
        arr2.sort()
        ans=[]
        for ele in arr1:
            ans.append(bisect_right(arr2,ele)) #It return rightmost element where we have to insert e so that arr2 remain sorted
    
        return ans
